packages/@glimmerx/babel-plugin-component-templates/index.js

```diff
@@ -94,6 +96,21 @@ module.exports = function(babel, options) {
     }));
   }
 
+  function maybeAddGlimmerInlinePrecompileImport(state, programPath) {
+    if (!state.glimmerInlinePrecompile) {
+      const hbsIdentifier = t.identifier('hbs');
+
+      const hbsImportDeclaration = t.importDeclaration(
+        [t.importDefaultSpecifier(hbsIdentifier)],
+        t.stringLiteral('glimmer-inline-precompile')
+      );
+
+      programPath.unshiftContainer('body', [hbsImportDeclaration]);
```

Comment:
you can use addDefault from @babel/helper-module-imports instead.

Reply:
Add default doesn't behave the way I want. `addDefault` will still produce a name like `import { hbs as _hbs } from "@babel/..."`

Reply:
On second though, it might work to use `addDefault`

This is the transform we would use:
https://github.com/glimmerjs/babel-plugin-glimmer-inline-precompile/blob/master/index.js

And this line looks for the importId:
https://github.com/glimmerjs/babel-plugin-glimmer-inline-precompile/blob/master/index.js#L29

I think we might be fine to use, it but I haven't validated that yet.

Reply:
Updated it to use `addDefault` and modified it such that new `_hbs` importId is used at the identifier of the taggedTemplateExpression